lib/commands/gesture.js

```diff
@@ -24,21 +24,96 @@ commands.click = async function (el) {
   }
 };
 
+function isSameGestures (gestures, candidates) {
+  if (gestures.length !== candidates.length) {
+    return false;
+  }
+  for (let i in gestures) {
```

Comment:
i also wonder if you could wrap this whole for block in a try/catch; if you get an exception anywhere in here it must mean they are different as well, right?